.github/workflows/test.yml

```diff
@@ -45,12 +57,27 @@ jobs:
         docker-images: true
         swap-storage: false
 
+    - if: matrix.os == 'ubuntu-latest' || matrix.os == 'ubuntu-24.04-arm'
+      name: "nix build"
+      run: |
+        nix build \
+          --print-build-logs \
+          --builders "" \
+          --max-jobs 2 \
```

Comment:
```suggestion
          --max-jobs auto \
```
maybe better?

Reply:
Note that I'm using the remote-store building instructions:
https://github.com/nixbuild/nixbuild-action?tab=readme-ov-file#remote-store-building